glue/viewers/image/qt/profile_viewer_tool.py

```diff
@@ -53,28 +53,3 @@ def activate(self):
         if not any_added:
             profile_viewer.close()
             return
-
-        # If the reference data for the current image viewer is in the profile
-        # viewer, we make sure that it is used as the reference data there too
-        if self.viewer.state.reference_data in profile_viewer._layer_artist_container:
-
-            profile_viewer.state.reference_data = self.viewer.state.reference_data
-
-            # We now pick an attribute in the profile viewer that is one of the ones
-            # with a slider in the image viewer. Note that the attribute viewer may
-            # be a pixel attribute or world attribute depending on what information
-            # is available in the coordinates, so we need to be careful about that.
-
-            x_att = profile_viewer.state.x_att
-            reference_data = self.viewer.state.reference_data
-
-            if isinstance(profile_viewer.state.x_att, PixelComponentID):
-                for att in reference_data.pixel_component_ids:
-                    if att is not self.viewer.state.x_att and att is not self.viewer.state.y_att:
-                        if att is not profile_viewer.state.x_att:
-                            profile_viewer.state.x_att = att
-            else:
-                for att in reference_data.world_component_ids:
-                    if att is not self.viewer.state.x_att_world and att is not self.viewer.state.y_att_world:
-                        if att is not profile_viewer.state.x_att:
-                            profile_viewer.state.x_att = att
```

Comment:
Why was this removed?

Reply:
Will add this back... Think I was having problems with these lines before